helm/hpcc/templates/eclagent.yaml

```diff
@@ -50,6 +50,13 @@ data:
 {{- include "hpcc.addResources" (dict "me" .me.resources) | indent 12 }}
 {{- $appCmd := printf "%s %s %s _HPCC_ARGS_" $apptype (include "hpcc.configArg" .me) (include "hpcc.daliArg" .root) }}
 {{ include "hpcc.addCommandAndLifecycle" (. | merge (dict "command" $appCmd)) | indent 12 }}
+{{- if index .me "env" }}
```

Comment:
thought index was supposed to take an index # e.g. index .me 0
may need to be hasKey .me "env" here?
Or just :
{{- if .env }}